lib/ddtrace/contrib/que/integration.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/que/ext'
+require 'ddtrace/contrib/que/configuration/settings'
+require 'ddtrace/contrib/que/patcher'
+
+module Datadog
+  module Contrib
+    module Que
+      # Description of Que integration
+      class Integration
+        include Datadog::Contrib::Integration
+
+        MINIMUM_VERSION = Gem::Version.new('1.0.0.beta4')
```

Comment:
In case we change the minimum supported version, we should update this line.

Reply:
updated to 1.0.0.beta2 to match the rest.